src/components/NavigationBar/NavigationBar.tsx

```diff
@@ -0,0 +1,40 @@
+import imgLogo from 'assets/logo.png';
+import { ReactComponent as Paw } from 'assets/paw-solid.svg';
+export const NavigationBar = () => {
+  return (
+    <header className="header">
+      <div className="header__logo">
+        <img className="header__img" src={imgLogo} alt="logo" />
+        <div className="header__title">Puppylog</div>
+      </div>
+      <nav>
+        <ul>
+          <li className="header__link">
+            <div>
+              <Paw className="header__icon" />
+              About Us
+            </div>
+          </li>
+          <li className="header__link">
+            <div>
+              <Paw className="header__icon" />
+              Contact
+            </div>
+          </li>
+          <li className="header__link">
+            <div>
+              <Paw className="header__icon" />
+              Sign Up
+            </div>
+          </li>
```

Comment:
you repeat the same part of code, 
I suggest to create folder names "constants" in src folder, create  file navigationData.ts and inside create array of objects like
```js
export const navigationData = [
   {
      id: 0,
      text: "About us",
   },
   {
      id: 1,
      text: "Contact"
   }
]
```
and then inside this component import this array and use .map()

Reply:
interface for that one will be 
```js
export interface InavData {
   id: number,
   text: string
}

export interface InavDatas extends Array<InavData > {}
```

Reply:
If navBar will be approved I add links to the navigation using interface